js/app.js

```diff
@@ -358,7 +358,7 @@ $(document).ready(function () {
           else {
             $(el).parent().attr('action', 'update');
           }
-          $(el).parent().addClass('uncommitted');
+          // $(el).closest('label-entry').addClass('uncommitted');
```

Comment:
What's the rational of:
- Commenting out this line (inside label's code block)
- But not comment out its counterpart inside milestone's code block?

Reply:
They do not mirror each other since milestones do not share the label colorpicker.

- Line 361 addresses the `submit` action on the colorpicker, which in this case is what the `save` button does.
- Line 456 is responsible for monitoring input changes on milestone entries.

Reply:
Deleting Line 456 will cause milestone updates to not enable commits.

Reply:
Brilliant. I agree.

Reply:
Please give me a minute

Reply:
Hittin the hay. Happy to address any further issues when I'm up.

Reply:
The current partially fixes the problem. 

Please see the screenshot:

<img width="1310" alt="Screen Shot 2020-05-27 at 6 02 05 AM" src="https://user-images.githubusercontent.com/19198708/83011698-f11db200-9fdf-11ea-9ec3-43d06cb86c10.png">

Only `div.label-entry` should have the `action=update` attribute. `div.flex-container` shouldn't. 

The following code will fix the issue between Line 350 and Line 362 in `app.js`:

```
        if ($(el).val() === $(el).attr('data-orig-val')) {
          $(el).closest(".label-entry").attr('action', 'none');
          $(el).closest(".label-entry").removeClass('uncommitted');
        }
        else {
          if ($(el).closest(".label-entry").attr('new') === 'true') {
            $(el).closest(".label-entry").attr('action', 'create');
          }
          else {
            $(el).closest(".label-entry").attr('action', 'update');
          }
          $(el).closest(".label-entry").addClass('uncommitted');
        }
```

<img width="1296" alt="Screen Shot 2020-05-27 at 6 08 31 AM" src="https://user-images.githubusercontent.com/19198708/83012094-8faa1300-9fe0-11ea-897c-f5eace9a8f57.png">

@tangaw Could you make the amendment? Thanks.